src/restconfig/src/main/resources/applicationContext.xml

```diff
@@ -1,20 +1,23 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<beans xmlns="http://www.springframework.org/schema/beans"
-       xmlns:mvc="http://www.springframework.org/schema/mvc"
-       xmlns:context="http://www.springframework.org/schema/context"
-       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xsi:schemaLocation="
-        http://www.springframework.org/schema/beans
-        http://www.springframework.org/schema/beans/spring-beans.xsd
-        http://www.springframework.org/schema/mvc
-        http://www.springframework.org/schema/mvc/spring-mvc.xsd
-        http://www.springframework.org/schema/context
-        http://www.springframework.org/schema/context/spring-context.xsd">
-
-  <!-- <mvc:annotation-driven/> -->
-  <context:component-scan base-package="org.geoserver.catalog.rest"/>
-  <context:component-scan base-package="org.geoserver.security.rest"/>
-  <context:component-scan base-package="org.geoserver.catalog.rest"/>
-  <context:component-scan base-package="org.geoserver.service.rest"/>
-
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Looks like the file newlines have been changed, check your git or IDE configuration and avoid it.